src/settings-ui/Settings.UI/SettingsXAML/Controls/GPOInfoBarControl.xaml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Controls">
+    <Style TargetType="local:GPOInfoBarControl">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:GPOInfoBarControl">
+                    <StackPanel Orientation="Vertical">
+                        <ContentPresenter Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" />
```

Comment:
What about doing the is enabled thing here? Then we have a standard behavior if the warning is showing.

Reply:
Not sure if I follow?

Reply:
Can we handle `IsEnabled` (can setting be changed) here based on `ShowWarning` instead in the page's xaml code.This would lead to less code in the page an remove a failure source.